### ISSUE TITLE
Only the first node's position can be adjusted when a way is being created, not the next ones'

### ISSUE THREAD

**Issue by PontiacCZ** [MarcusWolschon/osmeditor4android#1329]
## Behaviour/Symptoms

When creating a new way, I can adjust the position of the first node by tapping inside the dashed circle and moving. When I place second node (and next ones), using the same operation the whole map is panned instead.

## Expected Behaviour

I expect to be able to reposition every freshly placed node surrounded by a dashed circle.

## How to recreate

1. Start creating a new way in Vespucci - unlock, long tap on the desired spot and tap the plus symbol.
2. You are in the creating path mode and your node is surrounded by a dashed circle.
3. Try to move this first node - it is possible (when you touch the screen outside the circle, you pan the map which is expected)
4. Create a second node of the path by tapping somewhere on the screen.
5. Try to move this second node - you pan the whole map instead.

## Crash dump submitted (no or yes + date)

No.

## Any other potentially relevant information

![image](https://user-images.githubusercontent.com/9153595/104906804-76494300-5984-11eb-821b-8a9d1697e9f0.png)

![image](https://user-images.githubusercontent.com/9153595/104906835-82350500-5984-11eb-9d6f-ae94ba2fe8ea.png)

**Comment by simonpoole** [comment]
This is related to https://github.com/MarcusWolschon/osmeditor4android/issues/1316 in the mean time I've addressed this, however it still wouldn't do what you want, because for all practical purposes it is treated just as a multiselect. That is both the "current" node and the way is moved (it just seems to be different for the first node because there is no way that could be moved).

So this potentially needs some more thinking what the desirable behaviour could be as both variants (moving everything, and moving just the node) are legit.

**Comment by PontiacCZ** [comment]
> both variants (moving everything, and moving just the node) are legit.

I agree that both variants are legit and they are possible when the first node is drawn (as I write in step 3). It just does not work with following created nodes - you are then only able to move the node.

**Comment by simonpoole** [comment]
https://github.com/MarcusWolschon/osmeditor4android/commit/d26b13e5310ebf89b195a94c805a05877bd662f4 allows dragging of the whole way without creating a new undo checkpoint.
